EMS/common-bundle/src/Common/Bridge/Core/CoreDataApiBridge.php

```diff
@@ -48,4 +49,10 @@ public function finalize(int $revisionId, array $rawData = []): string
     {
         return $this->dataApi->finalize($revisionId, $rawData);
     }
+
```

Comment:
What about an align method? With a from environment? And an unpublish off-course.